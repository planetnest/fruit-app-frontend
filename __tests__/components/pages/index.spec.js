import { shallow } from "enzyme";
import React from "react";
import Index from "../../../pages/index";

describe("Index Page", () => {
  it("should match snapshot", () => {
    const wrapper = shallow(<Index />);
    expect(wrapper).toMatchSnapshot();
  });
});
