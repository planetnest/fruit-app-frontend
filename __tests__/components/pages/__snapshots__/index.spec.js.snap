// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page should match snapshot 1`] = `ShallowWrapper {}`;
